src/ensembl/src/content/app/species/components/species-main-view/SpeciesMainView.scss

```diff
@@ -116,3 +113,9 @@
   }
 
 }
+
+.noData {
+  color: $black;
+  font-weight: $light;
+  line-height: 3;
```

Comment:
Instead of doing `line-height: 3` here, might I suggest setting a min-height (e.g. `min-height: 50px`) on the `collapsedContent` class? That would be a more general solution.